client/decode.go

```diff
@@ -55,6 +55,16 @@ func decodeResponse(params map[string]interface{}) operation {
 	code := params["253"].(int16)
 
 	switch code {
+	case 2:
+		operation := operationJoinResponse{}
+		// We have to pick out the params manually, because Decode fails because of unknown types
```

Comment:
Can you include a link to this issue: https://github.com/hmadison/photon_spectator/issues/2